### PR TITLE
Consider untagged 'clean' repo's 'snapshots' (#23)

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

**Comment by dwijnand** [comment]
This looks great. I'm just not sure about the name "distinctVersion".. 😕

Also, should this be considered a breaking change?

**Comment by raboof** [comment]
yeah Naming Is Hard... also that key would be more useful if it was really independent of dynver.

Strictly speaking this is a breaking change, though I don't suspect many users rely on this interpretation. Still can't hurt to release `2.0.0` right? :)

**Comment by dwijnand** [comment]
Yeah I think I prefer to release this as 2.0.0.

Been thinking of a good name; right now I'm thinking we should call it `isVersionStable`.

**Comment by raboof** [comment]
I think that works - of course 'stable' is ambiguous too, but we'll just have to be clear in the key description.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+project
```
Comment (dwijnand) [inline .gitignore]: Could you remove this? I don't want to ignore the project directory.

**Comment by raboof** [comment]
No problem, I'll have a look tonight!

File: .gitignore (hunk 1 above)
Comment (raboof) [inline .gitignore]: Hm will do, not sure what I was thinking

**Comment by dwijnand** [comment]
Thank you. I've changed the branch this PR targets in preparation :)

**Comment by raboof** [comment]
Rebased and added `notes/2.0.0.markdown` - might be a bit verbose, but should explain the rationale behind this change.

**Comment by raboof** [comment]
Happy to, thanks for maintaining the plugin!

**Pull request merged**